src/common/ops/path.js

```diff
@@ -38,6 +39,7 @@ export function unwrap(children, path) {
     node = children[findFirst(children, key)];
     if (!node || node.key > key) return undefined; // We lack knowledge.
     if (isRange(node)) return null; // This is known to be null.
+    if (node.path) return unwrap(tree, node.path.concat(path.slice(i + 1)));
```

Comment:
The "unwrap" function is basically graffy equivalent of lodash "get". This line adds support for following links while getting a path in an object.